### DISCUSSION TITLE
续订专业版账户支付成功，但未续订成功怎么回事

### DISCUSSION THREAD

**Discussion by JH7431** [getlantern/lantern-forum-cn#1051] [Q&A]
![e553459c21a5bc8746d3a4d6464f4fa](https://github.com/user-attachments/assets/019a5ded-fb52-4b28-8362-206240403d45)
![微信截图_20241125164722](https://github.com/user-attachments/assets/2c17e059-c8cf-400d-8910-f2f2b7196eca)

**Comment by MaxMKMax** [comment]
@JH7431 您好. 我已经帮您提交问题上去了. 谢谢.

**Reply by JH7431** [reply]
使用email登陆收不到验证码？

**Comment by greySkiesMarsh** [comment]
Hi @JH7431 

I've adjusted your account's expiration to reflect the purchase, and cleared out the devices previously authorized to your account. You should be able to log in normally.

**Reply by JH7431** [reply]
thanks but I have problem receiving the device linking pin on my email, how long doesn't it take to receive a pin? I've been waiting for like 24 hours

**Reply by JH7431** [reply]
I can't receive device linking pin, could you please check? Do I need a Lantern Pro activation code to log in?

**Reply by MaxMKMax** [reply]
@JH7431 I will help you to resubmit the case again to get the code. Yes you may need Pro activation code to log in if you are still a free account. 

This is the FAQ in mandarin about how to link the device and login, i hope it would be helpful. Thanks.
https://lantern.io/zh-Hans/faq
